packages/web3-eth-accounts/src/account.ts

```diff
@@ -53,10 +81,129 @@ export const privateKeyToAccount = (
 	const publicKeyString = `0x${publicKey.slice(2)}`;
```

Comment:
Can we add a comment about why `.slice(2)`?

Reply:
Unfortunately there are no comments on either [ethers](https://github.com/MaiaVictor/eth-lib/blob/da0971f5b09964d9c8449975fa87933f0c9fef35/lib/account.js#L25) or the library we were using before ([eth-lib](https://github.com/MaiaVictor/eth-lib/blob/da0971f5b09964d9c8449975fa87933f0c9fef35/lib/account.js#L25)). I can add these links as a comment to this line though

Reply:
Yea, can't find anything about either